### PR TITLE
solution

### PR DESCRIPTION
**Pull request by BodyaRespect**
### **[DEMO LINK](https://BodyaRespect.github.io/react_autocomplete/)**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,99 +1,120 @@
-import React from 'react';
+import React, {
+  useState,
+  useRef,
+  useMemo,
+  useCallback,
+  useEffect,
+} from 'react';
+import { debounce } from 'lodash';
```
Comment (volodymyr-soltys97) [inline src/App.tsx:8]: Fix it
image

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, your demo link doesn't work, you need to fix it
image

**Comment by BodyaRespect** [comment]
[DEMO LINK](https://BodyaRespect.github.io/react_autocomplete/)

**Comment by BodyaRespect** [comment]
[DEMO LINK](https://BodyaRespect.github.io/react_autocomplete/)

File: src/App.tsx (hunk 1 above)
Comment (BodyaRespect) [reply src/App.tsx:8]: already installed??
image

Comment (volodymyr-soltys97) [reply src/App.tsx:8]: Still not fixed, you need to push your code after installed lodash
image

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, the demo link still not working, you need to make the deploy and check the deploy link after
image

**Pull request closed without merging**